### PR TITLE
(fix) start page lets you know if you're on a mobile browser and tells y...

### PR DESCRIPTION
**Pull request by rmagee88**
...ou to get the app

### CODE REVIEW

File: client/core/chrome-detect.js [hunk 1]
```diff
@@ -29,6 +31,20 @@
...
+      // if the user is on mobile, direct them to our site to get the native clients
+      browserIsMobile = ( window.navigator.userAgent.match(/Android/i) ||
+                          window.navigator.userAgent.match(/webOS/i) ||
+                          window.navigator.userAgent.match(/iPhone/i) ||
+                          window.navigator.userAgent.match(/iPad/i) ||
+                          window.navigator.userAgent.match(/iPod/i) ||
+                          window.navigator.userAgent.match(/BlackBerry/i) ||
+                          window.navigator.userAgent.match(/Windows Phone/i) );
+
+      return browserIsMobile;
```
Comment (ajkitson) [inline client/core/chrome-detect.js:46]: Think we also need to check `!window.cordova` here.

Comment (rmagee88) [inline client/core/chrome-detect.js:46]: Good catch, I'll test that on mobile this morning-

**Comment by rmagee88** [comment]
@ajkitson fix #329

**Comment by ajkitson** [comment]
lgtm :ship:

**Pull request merged**